taskcluster/scripts/lint/update_buildconfig_from_gradle.py

```diff
@@ -18,51 +19,45 @@
 
 logger = logging.getLogger(__name__)
 
-CONFIGURATIONS_WITH_DEPENDENCIES = (
-    "api",
-    "compileOnly",
-    "implementation",
-    "testImplementation",
-)
```

Comment:
I removed the configurations introduced in https://github.com/mozilla-mobile/android-components/pull/8594 because it didn't show transitive dependencies.

For instance: `./gradlew --parallel dependencies :compose-engine:dependencies` shows `support-ktx` but none of these commands do:

 * `./gradlew --parallel dependencies :compose-engine:dependencies --configuration api`
 * `./gradlew --parallel dependencies :compose-engine:dependencies --configuration compileOnly`
 * `./gradlew --parallel dependencies :compose-engine:dependencies --configuration implementation`
 * `./gradlew --parallel dependencies :compose-engine:dependencies --configuration testImplementation`

We actually benefit from this simplification: this script runs now much faster since it only shells out once instead of 4 times.